pynamodb/settings.py

```diff
@@ -0,0 +1,41 @@
+import imp
+import os
+from os import getenv
+from botocore.vendored import requests
+import logging
+
+log = logging.getLogger(__name__)
+
+OVERRIDE_SETTINGS_PATH = getenv('PYNAMO_CONFIG', '/etc/pynamodb/settings_override.py')
+
+settings = {}
+
+DEFAULT_REQUEST_SESSION_HEADERS = dict()
+
+default_settings_dict = {}
+default_settings_dict['REQUEST_TIMEOUT_SECONDS'] = 25
+default_settings_dict['MAX_RETRY_ATTEMPTS'] = 3
+default_settings_dict['BASE_BACKOFF_MS'] = 25
+default_settings_dict['REQUEST_SESSION_HEADERS'] = DEFAULT_REQUEST_SESSION_HEADERS
+
+class RequestSessionWithHeaders(requests.Session):
```

Comment:
Removed the default values.

All the default values are now in one place constants.py